
package despacho.ws.servicios.expuestos;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.7.redhat-1
 * 2014-11-01T16:46:31.517-03:00
 * Generated source version: 2.7.7.redhat-1
 */

@WebFault(name = "ArticuloDesconocidoException", targetNamespace = "http://soap.interfaces.tpo.ia.fain.uade/")
public class ArticuloDesconocidoException_Exception extends java.lang.Exception {
    
    private despacho.ws.servicios.expuestos.ArticuloDesconocidoException articuloDesconocidoException;

    public ArticuloDesconocidoException_Exception() {
        super();
    }
    
    public ArticuloDesconocidoException_Exception(String message) {
        super(message);
    }
    
    public ArticuloDesconocidoException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public ArticuloDesconocidoException_Exception(String message, despacho.ws.servicios.expuestos.ArticuloDesconocidoException articuloDesconocidoException) {
        super(message);
        this.articuloDesconocidoException = articuloDesconocidoException;
    }

    public ArticuloDesconocidoException_Exception(String message, despacho.ws.servicios.expuestos.ArticuloDesconocidoException articuloDesconocidoException, Throwable cause) {
        super(message, cause);
        this.articuloDesconocidoException = articuloDesconocidoException;
    }

    public despacho.ws.servicios.expuestos.ArticuloDesconocidoException getFaultInfo() {
        return this.articuloDesconocidoException;
    }
}
